www/main/menu.js

```diff
@@ -330,6 +335,10 @@ module.exports = {
     $listOfShabadOptions.appendChild(hukamnamaButton);
     $listOfShabadOptions.appendChild(notificationButton);
 
+    isOnline().then(online => {
```

Comment:
Would the app need to be restarted to re-enabled the button if the user comes online? should be fine but just wondering if we should make that better.